meta-python/recipes-devtools/python/python-evdev_0.6.0.bb

```diff
@@ -4,15 +4,18 @@ SECTION = "devel/python"
 LICENSE = "BSD"
 LIC_FILES_CHKSUM = "file://LICENSE;md5=18debddbb3f52c661a129724a883a8e2"
 
+SRC_URI = "https://github.com/gvalkov/python-evdev/archive/v${PV}.tar.gz;downloadfilename=${PN}-${PV}.tar.gz \
```

Comment:
I do not trust github to maintain persistent tarballs. JaMa and I have both seen fetch failures over the years. That is why I used pypi.